README.md

```diff
@@ -293,165 +317,78 @@ export default MessageList = () => {
 
-Configure an instance of the [Auth service][] and pass it to the `Security` component.
+#### Example
 
 ```jsx
-// src/App.js
-
-import React, { Component } from 'react';
-import { Router, Route } from 'react-router-dom';
-import { AuthService, Security, LoginCallback, SecureRoute } from '@okta/okta-react';
-import Home from './Home';
-import Protected from './Protected';
+import { useHistory } from 'react-router-dom';
+import { OktaAuth } from '@okta/okta-auth-js';
 
-const authService = new AuthService({
+const oktaAuth = new OktaAuth({
   issuer: 'https://{yourOktaDomain}.com/oauth2/default',
   clientId: '{clientId}',
-  redirectUri: window.location.origin + '/implicit/callback',
+  redirectUri: 'window.location.origin' + '/login/callback'
 });
 
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <Security authService={authService} >
-          <Route path='/' exact={true} component={Home}/>
-          <SecureRoute path='/protected' component={Protected}/>
-          <Route path='/implicit/callback' component={LoginCallback} />
-        </Security>
-      </Router>
-    );
-  }
-}
+export default App = () => {
+  const history = useHistory();
 
-export default App;
+  const customAuthHandler = (oktaAuth) => {
+    // Redirect to the /login page that has a CustomLoginComponent
+    // This example is specific to React-Router
+    history.push('/login');
+  };
+
+  return (
+    <Security
+      oktaAuth={oktaAuth}
+      onAuthRequired={customAuthHandler}
+    >
+      <Route path='/login' component={CustomLoginComponent}>
+      {/* some routes here */}
+    </Security>
+  );
+};
 ```
 
 #### PKCE Example
 
-Assuming you have configured your application to allow the `Authorization code` grant type, simply pass `pkce=true` to the `Security` component. This will configure the [Auth service][] to perform PKCE flow for both login and token refresh.  
-
-```jsx
-
-class App extends Component {
-  render() {
-    return (
-      <Router>
-        <Security issuer='https://{yourOktaDomain}.com/oauth2/default'
-                  clientId='{clientId}'
-                  pkce={true}
-                  redirectUri={window.location.origin + '/implicit/callback'}>
-          <Router path='/login' component={CustomLoginComponent}>
-          {/* some routes here */}
-        </Security>
-      </Router>
-    );
-  }
-}
-```
+Assuming you have configured your application to allow the `Authorization code` grant type, you can implement the PKCE flow with the following steps:
 
-You may also configure an instance of the [Auth service][] directly and pass it to the Security component.
+- Initialize [oktaAuth](Okta Auth SDK) instance with `pkce=true` and pass it to the `Security` component.
+- add `/login/callback` route with [LoginCallback](#logincallback) component to handle login redirect from OKTA.
```

Comment:
In addition to the /implicit/callback vs /login/callback issue, we should be sure to mention that whatever they use needs to match the configuration of their app in Okta.

Reply:
Agree! I don't really have a strong opinion on which one to use, probably use `/login/callback` and mention that they need to match config in okta app can save us from some future changes.

Reply:
I think PKCE is worth a mention, but I agree that we can mention it...and point to the AuthJS docs.

Note, however, that I'm not very happy with the AuthJS docs.  We should improve those before we start removing too much from elsewhere and leaning on AuthJS docs.  An overhaul to focus on customer use-cases like Aaron did with the Widget is overdue, I think.

Reply:
looks like with so many repo/doc depend on auth-js readme, we probably want to maintain proper versions of readme for `auth-js`, so we don't end up with broken links everywhere.